src/encoded/region_indexer.py

```diff
@@ -79,7 +79,9 @@ def includeme(config):
     config.scan(__name__)
     config.add_route('_regionindexer_state', '/_regionindexer_state')
     registry = config.registry
-    registry['region'+INDEXER] = RegionIndexer(registry)
+    is_region_indexer = registry.settings.get('regionindexer')
```

Comment:
There are a number of processes running, this change only create a region indexer for the region indexer process.